docs/Configuration.md

```diff
@@ -50,8 +53,18 @@
 - **totalPriceStatus** - The status of the total price used. Defaults to `"FINAL"`.
 - **allowPrepaidCards** - Set to `false` if you don't support prepaid cards. Default: The prepaid card class is supported for the card networks specified.
 - **billingAddressRequired** - Set to `true` if you require a billing address. A billing address should only be requested if it's required to process the transaction.
+- **billingAddressParameters** - Set billing address parameters:
+
+  - **format** - Billing address format required to complete the transaction. _MIN_: Name, country code, and postal code (default)._FULL_: Name, street address, locality, region, country code, and postal code.
+  - **phoneNumberRequired** - Set to true if a phone number is required for the provided shipping address.
+
 - **emailRequired** - Set to `true` to request an email address.
 - **shippingAddressRequired** - Set to `true` to request a full shipping address.
+- **shippingAddressParameters** - Set shiping address parameters.
```

Comment:
Typo: "shiping" -> "shipping"

Reply:
Fixed in [9532e80](https://github.com/Adyen/adyen-react-native/pull/297/commits/9532e80dd10c9f43f8b43db67dbe84b52fc2a5d8)